sci-physics/vmc/vmc-1.0_p2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/vmc-project/${PN}.git"
+else
+	MY_PV=$(ver_rs 1-2 -)
+	SRC_URI="https://github.com/vmc-project/${PN}/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/${PN}-${MY_PV}"
+fi
+
+DESCRIPTION="The Virtual Monte Carlo core library."
+HOMEPAGE="https://vmc-project.github.io/"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="+c++11 c++14 c++17 doc"
+
+REQUIRED_USE="^^ ( c++11 c++14 c++17 )"
+
+RDEPEND="c++11? (
+		>=sci-physics/root-6.18:=[c++11,-vmc]
```

Comment:
Can probably be simplified to
```bash
RDEPEND=">=sci-physics/root-6.18:=[c++11?,c++14?,c++17?,-vmc]"
```

Reply:
Very nice! I've been looking for exactly this kind of syntax, but somehow managed to miss it.